LICENSE

```diff
@@ -1,7 +1,6 @@
+MIT License
 
-The MIT License (MIT)
-
-Copyright (c) 2019 Wes Todd <wes@wesleytodd.com>
+Copyright Node.js contributors. All rights reserved.
```

Comment:
@wesleytodd is this OK with you?

We still have to decide what we're going to do about the `AUTHORS` file and whether to link to it here, but that's a dicussion for https://github.com/nodejs/package-maintenance/issues/348.